tests/host_tools/cargo_build.py

```diff
@@ -7,7 +7,7 @@
 from pathlib import Path
 
 from framework import defs, utils
-from framework.defs import FC_WORKSPACE_DIR, FC_WORKSPACE_TARGET_DIR
+from framework.defs import FC_WORKSPACE_DIR
```

Comment:
I think the constant is not used anywhere else, could be dropped from defs.py

Reply:
Oh, neat! 

As an aside, since we synced offline to do the A/B-testing by compiling the binaries outside of pytest, this commit is actually not needed anymore, so we can just drop it altogether. I personally do not really see a use for it anymore.